src/rcc.rs

```diff
@@ -338,17 +338,15 @@ impl CFGR {
         assert!(pclk2 <= 72_000_000);
 
         // adjust flash wait states
-        unsafe {
-            acr.acr().modify(|_, w| {
-                w.latency().bits(if sysclk <= 24_000_000 {
-                    0b000
-                } else if sysclk <= 48_000_000 {
-                    0b001
-                } else {
-                    0b010
-                })
-            })
-        }
+        acr.acr().modify(|_, w| {
```

Comment:
All those magic numbers make me nervous.
I'm not sure if it helps that they are [documented but not explained in stm32f3](https://docs.rs/stm32f3/0.11.0/src/stm32f3/stm32f303/flash/acr.rs.html#16-23)